_includes/menu.html

```diff
@@ -38,7 +38,7 @@
                 {% assign latestRelease = releaseDetails[formattedLastVersion] %}
                 {% if latestRelease.stable %}
                   {% assign labelStyle1 = "label label-stable" %}
-                  {% assign labelText1 = "stable" %}
+                  {% assign labelText1 = "latest stable" %}
```

Comment:
In the case of more than one stable version, it will show the `latest stable` in all the versions.